e2e/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+var TRACE_API_KEY_TEST = 'api-key'
+var TRACE_SERVICE_NAME_TEST = 'service-name'
+
+var childProcessTest = require('./utils/childProcessTest')
+var test = require('tape')
```

Comment:
@peteyy because it generates tap output by default so it is very easy to report consistently in child processes